openrouteservice/src/test/java/org/heigit/ors/apitests/isochrones/ParamsTest.java

```diff
@@ -18,6 +18,7 @@
 import org.heigit.ors.apitests.common.VersionAnnotation;
 import org.json.JSONArray;
 import org.json.JSONObject;
+import org.junit.jupiter.api.Disabled;
```

Comment:
Seems to be unused.

Reply:
Oh, sure, the test is enabled now. Removed.